website/docs/docs/cloud/manage-access/about-access.md

```diff
@@ -8,142 +8,203 @@ pagination_prev: null
-group, we recommend using a 1:1 mapping to make administration as simple as
-possible. Consider using the same name for your dbt Cloud groups and your IdP
-groups.
+For more information on these license types, see [Seats & Users](/docs/cloud/manage-access/seats-and-users)
 
-:::
+### Permissions
+
+Permissions determine what a developer-licensed user can do in your dbt Cloud account. By default, members of the `Owner` and `Member` groups have full access to all areas and features. When you want to restrict access to features, assign users to groups with stricter permission sets. Keep in mind that if a user belongs to multiple groups, the most permissive group will take precedence.
+
+The permissions available depends on whether you're on an [Enterprise](/docs/cloud/manage-access/enterprise-permissions) or [self-service Team](/docs/cloud/manage-access/self-service-permissions) plan. Developer accounts only have a single user, so permissions aren't applicable.
+
+<Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/access-control/assign-group-permissions.png" width="60%" title="Example the permissions dropdown while editing an existing group." />
+
+Some permissions (those that don't grant full access, like admins) allow groups to be "assigned" to specific projects and environments only. Read about [environment-level permissions](/docs/cloud/manage-access/environment-permissions-setup) for more information on restricting environment access.
+
+<Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/access-control/environment-access-control.png" width="60%" title="Example the environment access control for a group with Git admin assigned." />
+
+## Role-based access control <Lifecycle status='enterprise' />
+
+Role-based access control (RBAC) allows you to grant users access to features and functionality based on their group membership. With this method, you can grant users varying access levels to different projects and environments. You can take access and security to the next level by integrating dbt Cloud with a third-party identity provider (IdP) to grant users access when they authenticate with your SSO or OAuth service.
+
+There are a few things you need to know before you configure RBAC for SSO users:
+- New SSO users join any groups with the **Add all new users by default** option enabled. By default, the `Everyone` and `Member` groups have this option enabled. Disable this option across all groups for the best RBAC experience.
+- You must have the appropriate SSO groups configured in the group details SSO section. If the SSO group name does not match _exactly_, users will not be placed in the group correctly. 
+  <Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/access-control/sso-window-details.png" width="60%" title="The Group details SSO section with a group configured." />
+- It's a good idea for your dbt Cloud group names to match the IdP group names.
```

Comment:
Can we just say this?
```suggestion
- Your dbt Cloud group names should match the IdP group names.
```

Reply:
I think "should" implies they have to and they don't. Only the SSO mapping must match exactly. How about
```suggestion
-dbt Labs recommends that your dbt Cloud group names match the IdP group names.
```

Reply:
Sounds great!